### DISCUSSION TITLE
Does support of controls with Right To Left

### DISCUSSION THREAD

**Discussion by mayost** [twinbasic/twinbasic#1807] [Q&A]
In VB6 there was support for controls with right to left such as:
textbox1.RightToLeft=true
listbox1.RightToLeft=true
combobox1.RightToLeft=true

Is there or is there a future that there will be support from the right to the left

Thanks

**Comment by fafalone** [comment]
Right-to-left is not currently implemented for most controls but is planned for before the first stable release (v1.0), so should be completed sometime over the next few months.
